src/index.ts

```diff
@@ -1,14 +1,49 @@
 import * as fs from 'fs';
 import * as path from 'path';
+import * as loadPostCssConfig from 'postcss-load-config';
 import * as ts_module from 'typescript/lib/tsserverlibrary';
+
 import { createMatchers } from './helpers/createMatchers';
 import { isCSSFn } from './helpers/cssExtensions';
 import { getDtsSnapshot } from './helpers/cssSnapshots';
 import { Options } from './options';
 
+import * as postcss from 'postcss';
+import * as postcssIcssSelectors from 'postcss-icss-selectors';
+
+const removePlugin = postcss.plugin('remove-mixins', () => (css) => {
+  css.walkRules((rule) => {
+    rule.walkAtRules((atRule) => {
+      if (atRule.name === 'mixin') {
+        atRule.remove();
+      }
+    });
+  });
+});
+
+function getPostCssConfig(dir: string) {
+  try {
+    return loadPostCssConfig.sync({}, dir);
+  } catch (error) {
+    return { plugins: [] };
+  }
+}
+
 function init({ typescript: ts }: { typescript: typeof ts_module }) {
   let _isCSS: isCSSFn;
+
   function create(info: ts.server.PluginCreateInfo) {
+    const postcssConfig = getPostCssConfig(info.project.getCurrentDirectory());
+
+    const processor = postcss([
+      removePlugin(),
+      ...postcssConfig.plugins.filter(
+        // Postcss mixins plugin might be async and will break the postcss sync output.
+        (plugin) => !['postcss-mixins'].includes(plugin.postcssPlugin),
```

Comment:
Maybe it's better to just add a note to the docs here? As the plugin (from what I read) is only async when files are passed in.

Reply:
Yup! If we don't remove it for them we would have to pass something like `env: editor` to accommodate. If the mixins aren't filtered and they are async, the plugin just doesn't work.